_docs/tasks/traffic-management/ingress.md

```diff
@@ -214,11 +214,6 @@ The following are the known limitations of Istio ingress:
      kubectl get ingress secure-ingress -o wide
      ```
 
-     ```bash
-     NAME             HOSTS     ADDRESS                 PORTS     AGE
-     secure-ingress   *         130.211.10.121          80        1d
```

Comment:
why is this sample output removed?

Reply:
That part is added back. Thanks.

Reply:
This PR is for the ingress controller task only. Maybe we can update the installation with a more general solution in future work?